apps/pay/app/[username]/layout.tsx

```diff
@@ -27,10 +27,13 @@ type Props = {
 export default async function UsernameLayout({ children, params }: Props) {
   let response: ApolloQueryResult<AccountDefaultWalletsQuery> | { errorMessage: string }
   try {
-    response = await getClient().query<AccountDefaultWalletsQuery>({
-      query: AccountDefaultWalletsDocument,
-      variables: { username: params.username },
-    })
+    response = await apollo
+      .unAuthenticate()
```

Comment:
this sounds like a weird wording. is that supposed to be mean `non authenticated` queries?

Reply:
`unauthenticate` can easily be interpreted as the action to log out.